WMFModel/WMFArticleDataStore.h

```diff
@@ -0,0 +1,25 @@
+#import "WMFBaseDataStore.h"
+
+@class MWKSearchResult, MWKLocationSearchResult, MWKArticle, WMFFeedArticlePreview, WMFArticle;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface WMFArticleDataStore : NSObject
```

Comment:
Same goes for this class as the content store… could we just collapse into a single "store"?